dali/operators/reader/file_reader_op.cc

```diff
@@ -44,6 +44,15 @@ When traversing subdirectories, the labels are assigned consecutive numbers.)cod
 
 ``stick_to_shard`` and ``random_shuffle`` cannot be used when this argument is set to True.)code",
       false)
+  .AddOptionalArg<vector<string>>("files", R"(A list of file paths to read the data from.
+
+If ``file_root`` is provided, it is prepended to the file paths.).
+When using ``files``, the labels are taken from ``labels`` argument or, if it was not supplied,
+contain indices at which given file appeared in the ``files`` list.)", nullptr)
```

Comment:
I think we should specify in the doc that `files` and `file_list` cannot be specified at the same time instead of only throwing an error.

Reply:
I can add it, but really, I think that a user of intelligence enabling use of any programming language would figure that much.